### ISSUE TITLE
a few suggestions / requests ?

### ISSUE THREAD

**Issue by Forza101** [PackeTsar/meraki-cli#17]
Hi John, Couple of requests for you sir!

Could we create a shortcut command path when assigning as static IP address to a Meraki Device, currently its quite long see example below. 

meraki -j devices updateDeviceManagementInterface --serial "QXXX-XXXX-XXX" --wan1 \
'{
        "usingStaticIp": true,
        "staticIp": "0.0.0.0",
        "staticSubnetMask": "255.255.255.0",
        "staticGatewayIp": "0.0.0.0",
        "staticDns": [
            "1.1.1.1"
        ],
        "vlan": 666
}'

Would it also be possible to globally enable static IP addresses for all devices with in a given network?

**Comment by PackeTsar** [comment]
Yes I think it would make sense to some custom commands into Meraki-CLI to make common tasks easier. Setting device managements IPs is a perfect example.

Right now my best idea is to add a new command type, maybe call it `extra`. All the extended CLI commands would go under that type. So setting a device management IP would look like `meraki extra setDeviceManagementIp --serial XXXX-XXXX-XXXX --ip 10.0.0.10/24 ...`
